### PR TITLE
Add support for numerical and color properties to PropertySelect

### PR DESCRIPTION
**Pull request by hobinjk**
- [x] Investigate issue where rules appear to be duplicated
- [x] Clean up UI

### CODE REVIEW

**Comment by hobinjk** [comment]
UI now looks like this: 
screen shot 2018-04-02 at 11 20 39 am

**Comment by mrstegeman** [comment]
There seem to be some CSS issues:
screen shot 2018-04-05 at 08 35 52

**Comment by hobinjk** [comment]
numberUnitProperty and its siblings are a bit too long to reasonably display but this is the best I've come up with so far:
screen shot 2018-04-05 at 12 27 02 pm

File: static/js/rules/PropertySelect.js [hunk 1]
```diff
@@ -53,7 +53,10 @@ PropertySelect.prototype.addOption = function(name, ruleFragment, selected) {
     elt.classList.add('selected');
   }
   elt.dataset.ruleFragment = JSON.stringify(ruleFragment);
-  elt.textContent = name;
+  const nameElt = document.createElement('span');
+  nameElt.classList.add('property-select-name');
+  nameElt.textContent = name;
```
Comment (mrstegeman) [inline static/js/rules/PropertySelect.js]: If you set the `title` attribute here, we'll get some hover text. At least then you can differentiate the properties.

Comment (hobinjk) [reply static/js/rules/PropertySelect.js]: Oh, nice idea! I'll add it

**Pull request merged**